ui/src/dashboards/containers/DashboardsPage.js

```diff
@@ -0,0 +1,77 @@
+import React from 'react';
+
+import {getDashboards} from '../apis';
+
+const DashboardsPage = React.createClass({
+  getInitialState() {
+    return {
+      dashboards: [],
+    };
+  },
+
+  componentDidMount() {
+    getDashboards().then((dashboards) => {
+      this.setState({
+        dashboards,
+      });
+    });
+  },
+
+  render() {
+    let tableHeader;
+    if (this.state.dashboards.length === 0) {
+      tableHeader = "Loading Dashboards...";
+    } else {
+      tableHeader = `${this.state.dashboards.length} Dashboards`;
```

Comment:
`this.state.dashboards` is used 3 times in this function. OCD tells me that `const {dashboards} = this.state;` at the top would be good